nix/shell.nix

```diff
@@ -93,6 +93,7 @@ in pkgs.mkShell {
     redocly
     ripgrep
     rsync
+    rust-parallel
```

Comment:
rust-parallel doesn't interleave the output so it's still just as readable as before